saga-web/src/main/resources/saga-frontend/src/app/@theme/components/switcher/switcher.component.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * @license
  * Copyright Akveo. All Rights Reserved.
- * Licensed under the MIT License. See License.txt in the project root for license information.
+ * Licensed under the MIT License. See licenses/LICENSE-ngxadmin for license information.
```

Comment:
I don't think the reference of LICENSE is clean enough.

Reply:
@WillemJiang lets discuss how best we can do this. I will make the changes as needed in the next PR.